numpy/__init__.pyi

```diff
@@ -3387,13 +3387,13 @@ class ufunc:
     # raise a ValueError ufuncs with that don't accept two input
     # arguments and return one output argument. Because of that we
     # can't type them very precisely.
-    def reduce(self, /, *args: Any, **kwargs: Any) -> NoReturn | Any: ...
-    def accumulate(self, /, *args: Any, **kwargs: Any) -> NoReturn | NDArray[Any]: ...
-    def reduceat(self, /, *args: Any, **kwargs: Any) -> NoReturn | NDArray[Any]: ...
-    def outer(self, *args: Any, **kwargs: Any) -> NoReturn | Any: ...
+    def reduce(self, /, *args: Any, **kwargs: Any) -> Any: ...
+    def accumulate(self, /, *args: Any, **kwargs: Any) -> NDArray[Any]: ...
+    def reduceat(self, /, *args: Any, **kwargs: Any) -> NDArray[Any]: ...
+    def outer(self, *args: Any, **kwargs: Any) -> Any: ...
```

Comment:
Hmm, NoReturn seems odd here, makes me wonder why it was there.

Reply:
@jorenham Thoughts?

Reply:
`NoReturn | T` is equivalent to `T`, as `NoReturn` is the "bottom type" (just like `Never`). You removing it makes sense 👌🏻